srcpkgs/cross-aarch64-linux-gnu/template

```diff
@@ -6,11 +6,11 @@ _glibc_patchver="72-g0f90d6204d"
 _linux_version=5.10.4
 pkgname=cross-aarch64-linux-gnu
 version=0.35
-revision=3
+revision=4
 build_style=void-cross
 configure_args="--with-arch=armv8-a"
 hostmakedepends="texinfo tar gcc-objc gcc-go flex perl python3"
-makedepends="isl15-devel libmpc-devel gmp-devel mpfr-devel
+makedepends="isl-devel libmpc-devel pkg-config gmp-devel mpfr-devel
```

Comment:
pkg-config should be in hostmakedepends.

Reply:
thanks